cob_cartesian_controller/action/CartesianController.action

```diff
@@ -11,3 +10,5 @@ bool success
 string message
 ---
 # Define a feedback message
+bool success
```

Comment:
In feedback I expect to get information about the current state, so process might be interesting or a bool "finished" might be interesting. But "success" should tell one about the final state!